src/category/category.controller.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Patch,
+  Param,
+  Delete,
+  UseGuards,
+  ParseUUIDPipe,
+} from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+
+import { JwtAuthGuard } from '@auth/guards/jwt-auth.guard';
+import { CategoryService } from './category.service';
+import { CreateCategoryDto } from './dto/create-category.dto';
+import { UpdateCategoryDto } from './dto/update-category.dto';
+import { CategoryDescription } from './decorators/category-description.decorator';
+
+@ApiTags('Category')
+@Controller('category')
+export class CategoryController {
```

Comment:
The whole controller should jwt protected. You can put it a a controller lever, instead of method level

Reply:
you are right